django_app/redbox_app/redbox_core/views.py

```diff
@@ -99,21 +98,21 @@ def upload_view(request):
             api = CoreApiClient(host=settings.CORE_API_HOST, port=settings.CORE_API_PORT)
 
             try:
-                file = File.objects.create(
-                    processing_status=ProcessingStatusEnum.uploaded.value,
-                    user=request.user,
-                    original_file=uploaded_file,
-                    original_file_name=uploaded_file.name,
-                )
-                file.save()
-                # TODO: update improved File object with elastic uuid
-            except ValueError as value_error:
-                errors.append(value_error.args[0])
-
-            try:
-                api.upload_file(settings.BUCKET_NAME, uploaded_file.name, request.user)
+                upload_file_response = api.upload_file(settings.BUCKET_NAME, uploaded_file.name, request.user)
```

Comment:
`api.upload_file` doesnt really upload a file it ingests an existing file that has already been uploaded (this should really be renamed!), so this must be called after `file.save()` ?

Reply:
Oh, I misunderstood. I'll look into that.

Reply:
Ah - we'll need to discuss. If we haven't hit the upload file endpoint, we don't know the core API UUID.

Reply:
ah good point